CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
-13.6.0
+14.0.0
 ------
-* Generate an optional unauthenticated home_controller and authenticated products_controller using the `--with-session-token` flag to use JWT session tokens
+* Bump gemspec ruby dependency to 2.5
+* Bump elliptic from 6.5.1 to 6.5.3
```

Comment:
I'm not sure we need to include the elliptic bump on the changelog. Also, we should probably reword the other message slightly indicate more clearly that Ruby 2.4 is no longer supported by this gem.

Reply:
See the latest changelog?